internal-dns/types/src/names.rs

```diff
@@ -35,6 +35,12 @@ pub enum ServiceName {
     ///
     /// NOTE: This is used for either single-node or a replicated cluster.
     ClickhouseNative,
+    /// The native TCP interface to a ClickHouse server.
+    ///
+    /// NOTE: This is a temporary service name for replicated cluster.
```

Comment:
I'm not sure this is temporary necessarily. We may end up keeping this name.

Related, I think we need to change the comment above `ClickhouseNative` to say that it only points to single-node servers.